docs/tutorials/initializedirtyimage.py

```diff
@@ -0,0 +1,190 @@
+# -*- coding: utf-8 -*-
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.11.2
+#   kernelspec:
+#     display_name: Python 3
+#     language: python
+#     name: python3
+# ---
+
+# + nbsphinx="hidden"
+# %matplotlib inline
+
+# + nbsphinx="hidden"
+# %run notebook_setup
+# -
+
+# # Not completely done, still have to do some links do some fleshing out, just want to get it here to make sure there isn't anything critically wrong.
+#
+# ## Initializing Model with the Dirty Image
+#
+# In this tutorial we will be going over how to optimize the model based off of the dirty image. It has a low loss value (maximal likelihood) and so this allows the optimization to be much quicker. The problem with the dirty image is that certain model paramterizations enforce image positivity while the dirty image usually contains regions of negative flux. So in this tutorial we will be optimizing the model to be similar to the dirty image (phrasing could be better here). We will also be going over saving and loading the model. After saving and loading it, it can be then optimized against the visibility data to complete it (though this will not be done in this tutorial). We will be using the dataset of the ALMA logo first used in the Gridding and Diagnostic Images tutorial.
```

Comment:
I would back up a little bit and provide more context. 

* Our RML algorithm is written as an optimization loop (see previous optimization tutorial)
* The number of iterations required to reach the optimal point will be fewer if we have a good starting point for the model parameters
* Currently we're just starting from a flat (uniform) image. The dirty image is a better starting point, since we've already established that it maximizes the likelihood function (if not priors)
* We've implemented an RML model that does not permit negative flux values. This is a good thing for astronomical image reconstruction, because real images don't have negative flux. 
* However, this presents a problem for using the dirty image as a starting point, since it contains negative values.
* This tutorial shows how to initialize the model parameters to reproduce the dirty image as closely as possible, under the constraint that all pixels must be positive